.gitmodules

```diff
@@ -27,4 +27,5 @@
 	url = https://github.com/mohanson/bn128.git
 [submodule "deps/godwoken-scripts"]
 	path = deps/godwoken-scripts
-	url = https://github.com/nervosnetwork/godwoken-scripts.git
+	url = https://github.com/jjyr/godwoken-scripts.git
```

Comment:
Please use master branch, the [PR](https://github.com/nervosnetwork/godwoken-scripts/pull/116) is merged

Reply:
done